WordPress/UITestsFoundation/Screens/Login/Unified/PasswordScreen.swift

```diff
@@ -38,6 +38,13 @@ public class PasswordScreen: ScreenObject {
         let continueButton = app.buttons["Continue Button"]
         continueButton.tap()
 
+        // The Simulator might as to save the password which, of course, we don't want to do
```

Comment:
While I'm waiting for Xcode upgrade, I'm leaving comments in non-review mode. The first one is a super-nitpick 🙂:

```suggestion
        // The Simulator might ask to save the password which, of course, we don't want to do
```